build/static/js/main.86cfdafa.js.LICENSE.txt

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license React
+ * react-dom.production.min.js
+ *
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * @license React
+ * react-jsx-runtime.production.min.js
+ *
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
```

Comment:
- [ ]  Great job so far 👏🏻, However, your application runs with errors.  I'm getting this error message 👇🏻 After running the `npm install` then `npm start`.  Please ensure that your app works as expected and runs without errors. 😉 👍🏻 

- Browser: 

![04b](https://user-images.githubusercontent.com/79058364/182744010-576ddd45-5c6c-415e-b711-1113ff668f86.png)

- Console:

![04c](https://user-images.githubusercontent.com/79058364/182744443-55c1307a-76d2-4bba-977c-3757747c57cb.png)

Reply:
Solved the dependency issue.